cmd/openshift-install/create.go

```diff
@@ -384,6 +384,7 @@ func addRouterCAToClusterCA(ctx context.Context, config *rest.Config, directory
 }
 
 func waitForBootstrapComplete(ctx context.Context, config *rest.Config) *clusterCreateError {
+	return newClientError(errors.New("BUGGIN'"))
```

Comment:
This should probably be something else ;-0)

Reply:
It was on purpose to force a bootstrap failure so we can see the bootstrap gather collection exercised in CI.